src/views/catalog/wizard/components/WizardFooter.tsx

```diff
@@ -21,15 +21,20 @@ import { clearSessionStorageVM, useWizardVMContext } from '../../utils/WizardVMC
 
 import { WizardNoBootModal } from './WizardNoBootModal';
 
-export const WizardFooter: React.FC<{ namespace: string }> = ({ namespace }) => {
+export const WizardFooter: FC<{ namespace: string }> = ({ namespace }) => {
   const history = useHistory();
   const { t } = useKubevirtTranslation();
-  const { tabsData, disableVmCreate, loaded: vmContextLoaded } = useWizardVMContext();
+  const {
+    tabsData,
+    updateTabsData,
+    disableVmCreate,
+    loaded: vmContextLoaded,
+  } = useWizardVMContext();
   const { isBootSourceAvailable, loaded: bootSourceLoaded } = useWizardSourceAvailable();
   const { createVM, error, loaded: vmCreateLoaded } = useWizardVmCreate();
   const { createModal } = useModal();
 
-  const [startVM, setStartVM] = React.useState(true);
+  const [startVM, setStartVM] = useState<boolean>(tabsData?.startVM ?? true);
```

Comment:
`tabsData?.startVM ?? true` will evaluate to true all the time:
tabsData?.startVM === true => true ?? true => true
tabsData?.startVM === false => false ?? true => true
you can remove the || true part as it will always should have the value carried from the drawer

Reply:
> tabsData?.startVM === false => false ?? true => true

Nope. Did you test this PR or just assuming? Because it returns `false` as expected. I did test this.

Note that we do need this "or" part (and using ?? is the best, checks for null and undefined, not for boolean 'false' value), because we need some default value, if `tabsData.startVM` will be undefined, as it can be, because you've suggested me to move `updateTabsData` to `onChange` (and so I did it), but this has consequences that if there's no any change made in the 1st step of creating a VM (catalog drawer), `tabsData.startVM` will be `undefined` for sure, and that's we don't want, it would break the UI. And so according to the original code, `true` was the default value, so I've used it in case that  `tabsData.startVM` is `undefined`.